app/components/forms/events/view/create-access-code.js

```diff
@@ -0,0 +1,100 @@
+import Ember from 'ember';
+import FormMixin from 'open-event-frontend/mixins/form';
+import moment from 'moment';
+
+const { Component, computed } = Ember;
+
+export default Component.extend(FormMixin, {
+  getValidationRules() {
+    return {
+      inline : true,
+      delay  : false,
+      on     : 'blur',
+
+      fields: {
+        accessCode: {
+          identifier : 'access_code',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.l10n.t('Please enter access code')
+            },
+            {
+              type  : 'regExp',
+              value : '^[a-zA-Z0-9_-]*$'
+            }
+          ]
+        },
+        numberOfAccessTickets: {
+          identifier : 'number_of_access_tickets',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.l10n.t('Please enter number of tickets')
+            },
+            {
+              type   : 'number',
+              prompt : this.l10n.t('Please enter proper number of tickets')
+            }
+          ]
+        },
+        allTicketTypes: {
+          identifier : 'all_ticket_types',
+          rules      : [
+            {
+              type   : 'checked',
+              prompt : this.l10n.t('Please select the appropriate choices')
+            }
+          ]
+        },
+        min: {
+          identifier : 'min',
+          optional   : true,
+          rules      : [
+            {
+              type   : 'number',
+              prompt : this.l10n.t('Please enter the proper number')
+            }
+          ]
+        },
+        max: {
+          identifier : 'max',
+          optional   : true,
+          rules      : [
+            {
+              type   : 'number',
+              prompt : this.l10n.t('Please enter the proper number')
+            }
+          ]
+        },
+        accessLink: {
+          identifier : 'access_link',
+          optional   : true,
+          rules      : [
+            {
+              type   : 'url',
+              prompt : this.l10n.t('Please enter a valid url')
+            }
+          ]
+        }
+      }
+    };
+  },
+  accessCodeStartDate: computed(function() {
```

Comment:
Why a computed property for this, please pass in all this in a object from the route's `model` function. Data down.